lib/tasks/create_ocr.rake

```diff
@@ -0,0 +1,12 @@
+namespace :scihist do
+  desc """
+    bundle exec rake scihist:create_or_remove_ocr
+  """
+  task :create_or_remove_ocr => :environment do
+    progress_bar = ProgressBar.create(total: Work.count, format: "%a %t: |%B| %R/s %c/%u %p%% %e")
+    Work.find_each(batch_size: 10) do |work|
+      WorkOcrCreatorRemover.new(work, do_now: true).process
+      progress_bar.increment
```

Comment:
When we run this as a cron job, we definitely don't want `do_now: true`.  We want the cron job to get out of there as soon as possible, and add any actual work to a bg queue.  

(Why? Because the bg workers can _scale_ and do things in parallel; and because long-running tasks are dangerous and can be interupted by platform restarts and such, a bunch of shorter bg jobs are much preferable). 

I am not convinced we need a `do_now` option at all. But I'm curious to hear use cases!  If we do, the rake task should definitely by default execute with bg jobs though. 

Also I don't think `do_now` is a good name for it. If it remains, maybe the option can be called something involving the word `foreground` (as opposed to` background`, alternately it might be something involving `background` set to `false`)  or `synchronous` (as opposed to asynchronouos), or something like that.

Reply:
OK - got rid of `do_now`.